bucket/nextdns.json

```diff
@@ -0,0 +1,31 @@
+{
+    "version": "1.7.1",
+    "description": "NextDNS CLI is a DNS53 to DNS-over-HTTPS (DoH) proxy with advanced capabilities to get the most out of the NextDNS service.",
+    "homepage": "https://github.com/nextdns/nextdns",
```

Comment:
```suggestion
    "description": "DNS53 to DNS-over-HTTPS (DoH) proxy with advanced capabilities to get the most out of the NextDNS service.",
    "homepage": "https://nextdns.io",
```

Reply:
I disagree with this change. This is not NextDNS, but just a CLI client made to be used with NextDNS, and in fact it can worj for other DNS servers too:

> Although the most advanced features will only work with NextDNS, this program can work as a client for any DoH provider or a mix of NextDNS + another DNS (split horizon).

Reply:
Okay I've found https://nextdns.io/cli which redirects to https://github.com/nextdns/nextdns/wiki

I'll use that.